### ISSUE TITLE
Error during Grafana-deployment

### ISSUE THREAD

**Issue by matthieu-robin** [aenix-io/cozystack#550]
We are on version 0.20.2.
Same issue on tenant-root and sub tenant.

Failed pod is stuck in initialization phase
Init container 'load-vm-ds-plugin' is in BackOff state
Pod events show continuous container restarts
Init container script reveals unauthenticated GitHub API calls
here is the logs
```
[ 2024-12-27T13:58:13.693 ]: tar: invalid tar magic
[ 2024-12-27T13:58:13.691 ]: + tar -xf /var/lib/grafana/plugins/vl-plugin.tar.gz -C /var/lib/grafana/plugins/
[ 2024-12-27T13:58:13.684 ]: 
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100     9  100     9    0     0     33      0 --:--:-- --:--:-- --:--:--    33
100     9  100     9    0     0     33      0 --:--:-- --:--:-- --:--:--    33
[ 2024-12-27T13:58:13.418 ]:                                  Dload  Upload   Total   Spent    Left  Speed
[ 2024-12-27T13:58:13.418 ]:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[ 2024-12-27T13:58:13.41 ]: + curl -L https://github.com/VictoriaMetrics/victorialogs-datasource/releases/download/v0.13.2/victorialogs-datasource-v0.13.2.tar.gz -o /var/lib/grafana/plugins/vl-plugin.tar.gz
[ 2024-12-27T13:58:13.41 ]: + ver=v0.13.2
[ 2024-12-27T13:58:13.107 ]: + head -1
[ 2024-12-27T13:58:13.106 ]: + grep -oE 'v[0-9]+\.[0-9]+\.[0-9]+'
[ 2024-12-27T13:58:13.106 ]: + curl -s https://api.github.com/repos/VictoriaMetrics/victorialogs-datasource/releases/latest
[ 2024-12-27T13:58:13.102 ]: + mkdir -p /var/lib/grafana/plugins/
```

**Comment by kvaps** [comment]
Hey @matthieu-robin this issue is fixed by https://github.com/aenix-io/cozystack/pull/538 and released in v0.21.0

Do you need to preform patch relese for v0.20 or you going to update v0.21?

**Comment by matthieu-robin** [comment]
We will upgrade :)

Thanks a lot
